jdaviz/app.py

```diff
@@ -943,21 +943,20 @@ def get_viewer_by_id(self, vid):
         """
         return self._viewer_store.get(vid)
 
-    def _get_wcs_from_subset(self, subset_state):
+    def _get_wcs_from_subset(self, subset_state, data=None):
         """ Usually WCS is subset.parent.coords, except special cubeviz case."""
-
+        parent_data = subset_state.attributes[0].parent if not data else self.data_collection[data]
```

Comment:
not sure if this is the exact right place for this, but can you raise an informative error if the data label you past is invalid or doesn't have a wcs?